coml/magics.py

```diff
@@ -295,6 +312,9 @@ def comlrun(self, line, cell):
         assert self.shell is not None
         output = None
         try:
+            # show as svg
+            if 'plt.show()' in cell:
```

Comment:
It can be potentially problematic when `plt.show()` is indented.

Better if we put this show as `coml.vis_utils.show()` and instruct CoML to generate code that uses `coml.vis_utils.show()` instead of `plt.show()` when generating the code (this might require modifying `prompt.py`).

Reply:
If generating code that uses `coml.vis_utils.show()` by modifying the prompt, it will affect the code that the user sees. I'm now directly replacing `plt.show()` with `show_svg(plt)`, which should also eliminate the indentation problem.

Reply:
I'm still feeling something might be wrong here. But let's keep it as is for now.